### DISCUSSION TITLE
Stay on top in multi-monitor and fullscreen

### DISCUSSION THREAD

**Discussion by Richard-L** [OpenRCT2/OpenRCT2#15681] [Feature requests]
Perhaps a bit specific, but in a multi-monitor setup and running ORCT2 in fullscreen on screen, can it stay on top even if a different screen is active? Effecitvely keep showing the game while you're typing elsewhere, how it would be in windowed mode.

**Comment by janisozaur** [comment]
I thought that was the purpose of this option:

![zaznaczenie_109](https://cloud.githubusercontent.com/assets/550290/14229418/d9e80646-f933-11e5-8bbf-73d31f80c98a.png)

**Comment by Richard-L** [comment]
Nope. Screen returns back to Desktop while game keeps running in the back.
In any case, on-top fullscreen display would be so useful, so please do consider!
